modules/appeals_api/app/services/appeals_api/pdf_construction/generator.rb

```diff
@@ -14,9 +14,12 @@ def generate
         @form_fill_path = fill_unique_form_fields
         #=> '{appeal.id}-tmp.pdf' OR '/tmp/#{appeal.id}-overlaid-form-fill-tmp.pdf'
 
-        @unstamped_path = insert_additional_pages
+        @all_pages_path = insert_additional_pages
```

Comment:
My only concern is the name of this variable, not sure if it's indicative of what the process is past this point.

Reply:
Yeah I wasn't sure what to call this either. It's not the final pdf, since the rebuild step might take place, but it does have all the pages for the final pdf...happy to rename to something else if you've got suggestions :)

Reply:
I like _all_pages_, since it's exactly what it is. However, I could get behind using a word like _prepared_ to indicate this collection of pages may be destined for more!